api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -167,21 +195,11 @@ private FirecloudWorkspaceId generateFirecloudWorkspaceId(String namespace, Stri
     Timestamp now = new Timestamp(clock.instant().toEpochMilli());
     // TODO: enforce data access level authorization
     org.pmiops.workbench.db.model.Workspace dbWorkspace = FROM_CLIENT_WORKSPACE.apply(workspace);
+    dbWorkspace.setFirecloudName(workspaceId.getWorkspaceName());
+    dbWorkspace.setWorkspaceNamespace(workspaceId.getWorkspaceNamespace());
     dbWorkspace.setCreator(userProvider.get());
     dbWorkspace.setCreationTime(now);
     dbWorkspace.setLastModifiedTime(now);
-    dbWorkspace.setFirecloudName(workspaceId.getWorkspaceName());
-    dbWorkspace.setWorkspaceNamespace(workspaceId.getWorkspaceNamespace());
-    dbWorkspace.setDiseaseFocusedResearch(workspace.getResearchPurpose().getDiseaseFocusedResearch());
```

Comment:
I'm confused -- why is this going away? Where is this being set now?

Reply:
I'm trying to move all of whatever we can into `FROM_CLIENT_WORKSPACE` just to match my understanding of what that is, but am also comfortable reverting that if my model is wrong.